site/content/docs/main/restore-hooks.md

```diff
@@ -181,6 +181,8 @@ Below are the annotations that can be added to a pod to specify exec restore hoo
     * How long to wait once execution begins. Defaults is 30 seconds. Optional.
 * `post.hook.restore.velero.io/wait-timeout`
     * How long to wait for a container to become ready. This should be long enough for the container to start plus any preceding hooks in the same container to complete. The wait timeout begins when the container is restored and may require time for the image to pull and volumes to mount. If not set the restore will wait indefinitely. Optional.
+* `post.hook.restore.velero.io/wait-for-ready`
+    * String representation of a boolean that ensure command will be launch when underlying container is fully [ready](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes). Any value except "true" will be considered as "false". Defaults is false. Optional.
```

Comment:
Please something on similar lines like "Even if one restore hook consists of `WaitForReady` flag as true, all the other hook execution will wait for `Ready` state.

Reply:
String representation of a boolean that ensure command will be launched when underlying container is fully [Ready](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes). Use with caution because if only one restore hook consists of `WaitForReady` flag as "true", all the other hook executions will wait for `Ready` state too. Any value except "true" will be considered as "false". Defaults is false. Optional.

... good for you @shubham-pampattiwar ?